data/euruko/euruko-2022/videos.yml

```
@@ -0,0 +1,148 @@
+
+    To watch with closed captions, view the livestream recording: https://www.youtube.com/watch?v=reVGR35H264&t=14985s
+  video_id: 2aVyTtxs0GU
+
+- title: Security Doesn’t Have To Be a Nightmare
+  raw_title: Security Doesn’t Have To Be a Nightmare by Wiktoria Dalach
+  speakers:
+    - Wiktoria Dalach
+  event_name: Euruko 2022
+  published_at: '2022-10-20'
+  description: |-
+    Security doesn’t have to be scary. From this talk, you will learn 5 tips that can almost immediately make your code base more secure.
+
+    To watch with closed captions, view the livestream recording: https://www.youtube.com/watch?v=reVGR35H264&t=19095s
+  video_id: l4uyAQZBwyY
+
+- title: Looking Into Peephole Optimizations
+  raw_title: Looking Into Peephole Optimizations by Maple Ong
+  speakers:
+    - Maple Ong
+  event_name: Euruko 2022
+  published_at: '2022-10-20'
+  description: |-
+    Let's learn about various peephole optimizations used to make bytecode generated by the Ruby compiler more performant. Do these small changes make any impact on the final runtime? Let's find out - experience reading bytecode is not needed!
+
+    To watch with closed captions, view the livestream recording: https://www.youtube.com/watch?v=reVGR35H264&t=22110s
+  video_id: gnX1o8GBed0
+
+- title: The Technical and Organizational Infrastructure of the Ruby Community
+  raw_title: The Technical and Organizational Infrastructure of the Ruby Community by Adarsh Pandit
+  speakers:
+    - Adarsh Pandit
+  event_name: Euruko 2022
+  published_at: '2022-10-20'
+  description: |-
+    This is a talk which reviews all of the things which enable Ruby developers to use a secure programming language with easy package management in a supportive community. Specifically we will be tracking money spent and code shipped.
+
+    To watch with closed captions, view the livestream recording: https://www.youtube.com/watch?v=m2qelRkp1CY&t=1650s
+  video_id: cNLscH0sGz4
+
+- title: From massive pull requests to trunk-based development with Ruby
+  raw_title: From massive pull requests to trunk-based development with Ruby by Vesa Vänskä
+  speakers:
+    - Vesa Vänskä
+  event_name: Euruko 2022
+  published_at: '2022-10-20'
+  description: |-
+    Talk about how to go from massive pull requests to trunk-based development with Ruby. Includes concrete examples of code, architecture, and workflows to implement smooth trunk-based development workflow.
+
+    To watch with closed captions, view the livestream recording: https://www.youtube.com/watch?v=m2qelRkp1CY&t=4980s
+  video_id: M76dNDySrNc
+
+- title: Lightning Talks & City Pitches
+  raw_title: Lightning talks & city pitches
+  speakers:
+    - Renato dos Santos Cerqueira
+    - Mohnish Jadwani
+    - Hans Schnedlitz
+    - Hana Harencarova
+    - Antti Merilehto
+    - Tomasz
```

```suggestion
    - Tomasz Cudziło
```
I think it is him This being said shouldn't we list only the speakers of the lighting talks?

- Renato dos Santos Cerqueira    
- Mohnish Jadwani
- Hans Schnedlitz
- Hana Harencarova

---

Yeah, not sure. Happy either way